benchmarks/asv.conf.json

```diff
@@ -20,7 +20,7 @@
 
     // The Pythons you'd like to test against.  If not provided, defaults
     // to the current version of Python used to run `asv`.
-    "pythons": ["3.6"],
+    //"pythons": ["3.6"],
```

Comment:
This is just an adjustment I made on my own computer.

Reply:
Uh huh. You shouldn't commit it. It needs to be removed from this PR.

Reply:
I should have been more clear. I made this adjustment on my own computer and didn't realize it ended up in the staged files for this commit.